### DISCUSSION TITLE
(Not a real issue) 2.7.1 -> 3.0.0 breaks PN532 RFID reader on I2C

### DISCUSSION THREAD

**Discussion by AnotherMatt92** [earlephilhower/arduino-pico#1271] [Q&A] [answered]
Im trying to read an RFID tag using an PN532 and the adafruit_pn532 library via I2C on the pi pico W

Example codes in the libraries and my own code works fine when using version 2.7.1 of this arduino pico board manager. 
But when I update to version 3.0.0 the code still compiles but nothing seems to happen on the bus and all responses just hang/ return 0

I dont change any default pins for i2c0 (gpio4/5 for sda/scl), even the examples just fail with their standard "Didn't find PN53x board" as no version data is return when called for

Any ideas on what im doing wrong here would be greatly appreciated?

**Comment by earlephilhower** [comment]
FWIW, according to https://github.com/earlephilhower/arduino-pico/commits/master/libraries/Wire there have been no changes since Jan 6, 2023

2.7.1 came out on Jan 14, 2023, so I don't think it's related to `Wire` changes.

**Reply by earlephilhower** [reply]
As an idea, try 2.7.3 just to minimize the delta from 2.7.x to 3.x.  You could also use the `git` version and test against each individual commit (or do a binary search) to see if there's really something amiss in the core.

**Reply by AnotherMatt92** [reply]
my apologies its not the board manager that was the issue, the adafruit PN532 was also updated and my library manager wasnt recompiling that properly until i changed board version too. i.e. i wasnt using the reverted library when i thought it was

**Comment by AnotherMatt92** [comment, marked as answer]
Not an actual issue, was libraries compile mismatches between file versions
